packages/shared/locales/en.json

```diff
@@ -1603,11 +1594,7 @@
         "jwt": "JSON web token",
         "internalTransaction": "Internal transaction",
         "coinType":"Coin type",
-        "custom": "Custom",
-        "evmAddress": "EVM address",
```

Comment:
filtering by `evmAddress` in the project, I have also seen an interface with the same name `IEvmAddress`, which is not used anywhere. Maybe we could remove it too

Reply:
Removed in the father PR